challenges/2021_cloud_base_height/04_eval_pytorch.ipynb

```diff
@@ -0,0 +1,1108 @@
+{
```

Comment:
<p>maybe split each function into separate cell?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MetOffice/data_science_cop/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
Done. I had kept them together in the same cell to justify my poor naming system, I used markdown to explain instead